flask-redis-centos7-atomicapp/Nulecule

```diff
@@ -0,0 +1,31 @@
+---
+specversion: 0.0.2
+id: flask_redis_nulecule
+
+metadata:
+    name: Flask Redis Nulecule App
+    appversion: 0.0.1
+    description: Atomic app for deploying Flask + Redis page visit counter
```

Comment:
"Atomic App of a simple light weight Flask and Redis app. This Flask app counts the number of time page is visited and stores the counter in Redis." sounds simple to me.